docs/architecture/translations.rst

```diff
@@ -37,21 +37,21 @@ Consider a product.
        translated = TranslationProxy()
 
 
-``Product`` has several properties, but only two of them are translatable - ``name`` and ``description``.
+The product has several properties, but want to translate just its ``name`` and ``description``.
 
-We've also set a ``translated`` property being an instance of ``TranslationProxy``.
+We've also set a ``translated`` property to an instance of ``TranslationProxy``.
 
-``ProductTranslation`` will be used to store our translated properties, it needs to have two fields:
+We will use ``ProductTranslation``  to store our translated properties, it requires two base fields:
 
 - ``language_code``
-    A language code that this translation correlates to
+    A language code that this translation correlates to.
 
 - ``product``
     ``ForeignKey`` relation to the translated object (in this case we named it *product*)
 
 ... and any other field you'd like to translate, in our example, we will use ``name`` and ``description``.
 
-.. warning:: ``TranslationProxy`` excepts ``related_name``, on the ``ForgeignKey`` relation to be set to ``translations``
+.. warning:: ``TranslationProxy`` except ``related_name``, on the ``ForeignKey`` relation to be set to ``translations``
```

Comment:
Did you mean "relation **has** to be set to"?

Reply:
Not really, but reworded the sentence a little